STLtoDATTranslator/Source Files/Point.cpp

```diff
@@ -7,4 +7,4 @@ class Point {
         double x, y, z;
         Point(double x, double y, double z) : x(x), y(y), z(z) {}
         ~Point() {}
```

Comment:
Point is the index for unique values and should be an integer

Reply:
Point has public data members which should not be accessed and can be made private. you can implement getter-setter methods to access it.